bigframes/ml/sql.py

```diff
@@ -318,6 +318,10 @@ def ml_evaluate(self, source_df: Optional[bpd.DataFrame] = None) -> str:
             return f"""SELECT * FROM ML.EVALUATE(MODEL `{self._model_name}`,
   ({source_sql}))"""
 
+    def ml_arima_coefficients(self) -> str:
```

Comment:
Could you help add the unit test for this method in https://github.com/googleapis/python-bigquery-dataframes/blob/main/tests/unit/ml/test_sql.py?

Reply:
+1 @SalemJorden please add such a test using the examples Ashley links here. Schedule a 1:1 with me to pair program if you need more support to get started.